specification/paths/SynchronizeIntegration.json

```diff
@@ -33,11 +33,13 @@
                   "shop_id": {
                     "type": "string",
                     "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
```

Comment:
we cannot use the Uuid schema here?

Reply:
The problem with the Uuid schema is that it does not include an example. And we do not want the same example uuid for all our resources. That would look weird in relationship examples.

Now I think about it, we could use the Uuid schema like this:
```
"allOf": [
  {
    "$ref": "#/components/schemas/Uuid"
  },
  {
    "example": "35eddf50-1d84-47a3-8479-6bfda729cd99"
  }
]
```